curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/-iterate-through-the-keys-of-an-object-with-a-for...in-statement.english.md

```diff
@@ -6,27 +6,62 @@ challengeType: 1
 
 ## Description
 <section id='description'>
-Sometimes you may need to iterate through all the keys within an object. This requires a specific syntax in JavaScript called a <dfn>for...in</dfn> statement. For our <code>users</code> object, this could look like:
-<blockquote>for (let user in users) {<br>&nbsp;&nbsp;console.log(user);<br>}<br><br>// logs:<br>Alan<br>Jeff<br>Sarah<br>Ryan</blockquote>
+Sometimes you may need to iterate through all the keys within an object. This requires a specific syntax in JavaScript called a <dfn>for...in</dfn> statement. In the example below, there is a <code>users</code> object and the loop iterates through it to display the user's name to the console.
+<blockquote>
```

Comment:
The only thing remaining from the first review was this - shouldn't we use `<pre><code>` to markup blocks of code?  I double-checked in [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/code#Notes) and that's what they recommend.

Reply:
I still think this should be discussed on an issue and not decide this large of a change (which would affect all challenges) on this PR.

Reply:
Sure, I'll put an issue up in a sec.